### DISCUSSION TITLE
Workaround for AI deck compatibility with Loco Positioning Deck

### DISCUSSION THREAD

**Discussion by joatjoses** [bitcraze/discussions#770] [Q&A]
Hello, I'm planning to use both AI deck and Loco Positioning Deck for my project and the compatibility matrix says that there are conflicts but a patch or workaround can be done to fix this. I am trying to look for solutions and a previous discussion pointed to the hyper demo (https://forum.bitcraze.io/viewtopic.php?t=4602) where they moved the LPS IRQ to IO2 and RST to IO4. Is this the same workaround for AI Deck and LPS?

**Comment by tobbeanton** [comment]
Please have a look at https://github.com/orgs/bitcraze/discussions/563

**Reply by joatjoses** [reply]
Hello! Thank you for referring this discussion. I have followed through the instructions and the console now shows that both decks have been found. However, it seems that the loco positioning deck is not working properly as it cannot find the anchors. It can detect the anchors when used alone but when used with AI deck, the deck LEDs are not on and it cannot detect the anchors.

**Reply by knmcguire** [reply]
Hi! Just checking since the previous discussion. Have you patched the locodeck with disconnecting GS1 and GS2 and bridged GS3 and GS4? If you look in the hyper demo blogpost, there is a picture under 'hardware' on how to do these patches with soldering.

https://www.bitcraze.io/2020/12/the-hyper-demo/

**Reply by knmcguire** [reply]
Ah sorry I saw that you already answered here: https://github.com/orgs/bitcraze/discussions/776. Let's continue over there.
